_posts/2015-12-11-License-and-You.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: 'Licenses for OSS Code'
+date: 2015-12-10T00:00:00.000Z
+comments: false
+categories: [ios, mobile, review, video, oss]
+author: orta
+---
+
+As a part of our internal Lunch and Learn series, I  gave a talk to our developers with an overview of the different types of source code licenses available.
+
+We always recommend MIT within Artsy, but understanding what the other ones are and why we made the choices we have done is valuable.
+
+Jump [to YouTube](https://www.youtube.com/watch?v=0r7JcN3Q_LY) for the video, or click more for a smaller inline preview, as well as all of the speakers notes.
+
+<!-- more -->
+
+{% youtube 0r7JcN3Q_LY %}
+
+### Licenses For Code
+
+Attn:*I am not a lawyer*, but I have to care about this from an app dev perspective, and a dependency manager perspective.
+
+Recommendation for further reading:
+
+* http://choosealicense.com
+* https://tldrlegal.com/
+
+We'll talk about 3 types of licenses:
+
+- Permissive: MIT/BSD - Jquery/ Rails/ Artsy OSS
+- Mostly Permissive: Apache, Mozilla - SVN / Firefox
+- Copyleft / Viral - Git/ Linux/ Wordpress
+
+
+### Common Components
+
+All Share one important thing:
+
+  * YOU MAY USE THIS
+  * NO WARRANTY
+  * DISCLAIMER OF LIABILITY
+
+Which is basically the crux of how we can all actually use and write OSS without ending up bankrupt. E.g. when a major component of your infrastructure "fails" - it's not the fault of the author. Example: [Shellshock](https://en.wikipedia.org/wiki/Shellshock_(software_bug)) - no-one tried suing the maintainers of Bash.
+
+### Public Domain
```

Comment:
The term "public domain" (at least in America) has [a very specific legal definition](https://en.wikipedia.org/wiki/Public_domain), which I'm not sure lines up with what you're trying to say:

> Works in the public domain are those whose intellectual property rights have expired, have been forfeited, or are inapplicable.

What I think you're referring to is the automatic copyright applied to any creative work. It could be worth using a term other than "public domain" to avoid confusion.